lib/plds_web/endpoint.ex

```diff
@@ -24,7 +28,7 @@ defmodule PLDSWeb.Endpoint do
     cookie_key: "request_logger"
 
   plug Plug.RequestId
-  plug Plug.Telemetry, event_prefix: [:phoenix, :endpoint]
+  plug Plug.Telemetry, event_prefix: [:plds, :endpoint]
```

Comment:
I think we can remove a lot of those plugs. No need for Static, RequestId... maybe not even Plug.Parsers.